testkit/src/main/java/org/apache/calcite/test/QuidemTest.java

```diff
@@ -187,6 +209,57 @@ protected void checkRun(String path) throws Exception {
     }
   }
 
+  private void updatePlanner(RelOptPlanner planner, String value) {
+    List<RelOptRule> rulesAdd = new ArrayList<>();
+    List<RelOptRule> rulesRemove = new ArrayList<>();
+    parseRules(value, rulesAdd, rulesRemove);
+    rulesAdd.forEach(planner::addRule);
+    rulesRemove.forEach(planner::removeRule);
```

Comment:
I don't know if the order of rules in the planner matters.
but just in case, perhaps it's safer to first remove, then add